src/main/java/com/vw/ipppdiffer/exception/UnknownParseException.java

```diff
@@ -0,0 +1,14 @@
+package com.vw.ipppdiffer.exception;
+
+
+import org.springframework.expression.ParseException;
+
+public class UnknownParseException extends ParseException {
+
```

Comment:
Can we rename to not contain the Unknown work? :D

Reply:
We can name it like DifferServiceParseException

Reply:
I'll rename it to DifferParseException because it should not contain the service name in it.